sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -172,19 +183,24 @@ def begin_recognize_content_from_url(self, url, **kwargs):
 
     @distributed_trace
     def begin_recognize_custom_forms(self, model_id, stream, **kwargs):
-        # type: (str, IO[bytes], Any) -> LROPoller
-        """Analyze Form.
+        # type: (str, Union[bytes, IO[bytes]], Any) -> LROPoller
```

Comment:
I'd add somewhere in the docstring that the form should be of the same type as the forms the custom model was trained with